<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1314</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(179, 191, 213);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="12" column="0" colspan="2">
       <widget class="QLabel" name="Calculatorslbl">
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(15, 82, 155);
font-weight: bold;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; text-decoration: underline;&quot;&gt;Calculators&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="20" column="0" colspan="2">
       <widget class="QLabel" name="DataBaseLbl">
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(15, 82, 155);
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; text-decoration: underline;&quot;&gt;Databases&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="QPushButton" name="BeamCall">
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Beam Calculator</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Python Guidebook</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="NewProjectBot">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>New Project</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="2">
       <widget class="QPushButton" name="CAMCall">
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>CAM Calculator</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QFrame" name="frame_8">
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(15, 82, 155);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="19" column="0" colspan="2">
       <widget class="QFrame" name="frame_3">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(15, 82, 155);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="UsefulToolslbl">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(15, 82, 155);
font-weight: bold;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; text-decoration: underline;&quot;&gt;Useful Tools&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="OpenProjectBot">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Open Project</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(15, 82, 155);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="22" column="0" colspan="2">
       <widget class="QPushButton" name="KBECall">
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>KBE</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QPushButton" name="ODRIVECall">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>LabView ODRIVE Controller</string>
        </property>
       </widget>
      </item>
      <item row="26" column="3">
       <widget class="QLabel" name="ActiveFile">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Active Project File: No Project Selected</string>
        </property>
       </widget>
      </item>
      <item row="21" column="0" colspan="2">
       <widget class="QPushButton" name="MaterialCall">
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Materials</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QFrame" name="frame_4">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(15, 82, 155);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Server/Database Guidebook</string>
        </property>
       </widget>
      </item>
      <item row="14" column="5">
       <widget class="QPushButton" name="ImportData">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Import data</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>FEMAP NASTRAN Guidebook</string>
        </property>
       </widget>
      </item>
      <item row="26" column="6">
       <widget class="QPushButton" name="SaveNotes">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Save Notes</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" rowspan="23">
       <widget class="QStackedWidget" name="MainStack">
        <property name="minimumSize">
         <size>
          <width>870</width>
          <height>638</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>870</width>
          <height>638</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(161, 162, 164);
border-radius: 5px
</string>
        </property>
        <widget class="QWidget" name="page"/>
        <widget class="QWidget" name="page_2"/>
       </widget>
      </item>
      <item row="0" column="4" rowspan="27">
       <widget class="QFrame" name="frame_6">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(15, 82, 155);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::VLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="27">
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(15, 82, 155);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::VLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="ProjectTitle">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(15, 82, 155);
font-weight: bold;
</string>
        </property>
        <property name="text">
         <string>No Project Selected</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="3" colspan="2">
       <widget class="QLabel" name="SierraTech">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Images/Media/SierraTechOG.png&quot;width=&quot;225&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="25" column="3">
       <widget class="QFrame" name="frame_5">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(15, 82, 155);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="25" column="5">
       <widget class="QPushButton" name="LoadNotes">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Load Notes</string>
        </property>
       </widget>
      </item>
      <item row="25" column="6">
       <widget class="QPushButton" name="SaveNotesAs">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Save Notes As</string>
        </property>
       </widget>
      </item>
      <item row="26" column="5">
       <widget class="QPushButton" name="ClearNotes">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Clear Notes</string>
        </property>
       </widget>
      </item>
      <item row="14" column="6">
       <widget class="QPushButton" name="ClearData">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Clear Data</string>
        </property>
       </widget>
      </item>
      <item row="15" column="5" colspan="2">
       <widget class="QPushButton" name="RunData">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 230, 232);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Run Data</string>
        </property>
       </widget>
      </item>
      <item row="17" column="5" colspan="2">
       <widget class="QLabel" name="noteslbl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>175</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(15, 82, 155);
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; text-decoration: underline;&quot;&gt;Notes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="18" column="5" rowspan="7" colspan="2">
       <widget class="QTextEdit" name="Notes">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(161, 162, 164);
background-color: rgb(229, 230, 232);
border-radius: 5px;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="5" rowspan="12" colspan="2">
       <widget class="QLabel" name="Metadata">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(161, 162, 164);
color: rgb(0, 0, 0);
border-radius: 5px</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="5" rowspan="2" colspan="2">
       <widget class="QLabel" name="Metadatalbl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(15, 82, 155);
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; text-decoration: underline;&quot;&gt;Metadata&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="16" column="5" colspan="2">
       <widget class="QFrame" name="frame_7">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(15, 82, 155);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="MainResources.qrc"/>
 </resources>
 <connections/>
</ui>
